prime-dotnet-webapi/Migrations/20211217163513_ChangedBCSCReference.cs

```diff
@@ -0,0 +1,308 @@
Net,\r\n      either for you or for the Practitioner(s), and that you must comply with any such further limits and conditions.\r\n    </p>\r\n\r\n  </li>\r\n  <li>\r\n\r\n    <p class=\"bold underline\">\r\n      How to Notify the Province\r\n    </p>\r\n\r\n    <p>\r\n      Notice to the Province may be sent in writing to:\r\n    </p>\r\n\r\n    <address>\r\n      Director, Information and PharmaNet Development<br>\r\n      Ministry of Health<br>\r\n      PO Box 9652, STN PROV GOVT<br>\r\n      Victoria, BC V8W 9P4<br>\r\n\r\n      <br>\r\n\r\n      <a href=\"mailto:PRIMESupport@gov.bc.ca\">PRIMESupport@gov.bc.ca</a>\r\n    </address>\r\n\r\n  </li>\r\n  <li>\r\n\r\n    <p class=\"bold underline\">\r\n      Province may modify these terms\r\n    </p>\r\n\r\n    <p>\r\n      The Province may amend these terms, including this section, at any time in its sole discretion:\r\n    </p>\r\n\r\n    <ol type=\"i\">\r\n      <li>\r\n        by written notice to you, in which case the amendment will become effective upon the later of (A) the date\r\n        notice of the amendment is first delivered to you, or (B) the effective date of the amendment specified by the\r\n        Province, if any; or\r\n      </li>\r\n      <li>\r\n        by publishing notice of any such amendment in the PharmaCare Newsletter, in which case the notice will specify\r\n        the effective date of the amendment, which date will be at least thirty (30) days after the date that the\r\n        PharmaCare Newsletter containing the notice is first published.\r\n      </li>\r\n    </ol>\r\n\r\n    <p>\r\n      If you do not agree with any amendment for which notice has been provided by the Province in accordance with (i)\r\n      or (ii) above, you must promptly (and in any event before the effective date) cease all access or use of\r\n      PharmaNet.\r\n    </p>\r\n\r\n    <p>\r\n      Any written notice to you under (i) above will be in writing and delivered by the Province to you using any of the\r\n      contact mechanisms identified by you in PRIME, including by mail to a specified postal address, email to a\r\n      specified email address or text message to a specified cell phone number. You may be required to click a URL link\r\n      or log into PRIME to receive the contents of any such notice.\r\n    </p>\r\n\r\n  </li>\r\n  {$lcPlaceholder}\r\n  <li>\r\n\r\n    <p class=\"bold underline\">\r\n      Governing Law\r\n    </p>\r\n\r\n    <p>\r\n      These terms will be governed by and will be construed and interpreted in accordance with the laws of British\r\n      Columbia and the laws of Canada applicable therein.\r\n    </p>\r\n\r\n    <p>\r\n      Unless otherwise specified, a reference to a statute or regulation by name means the statute or regulation of\r\n      British Columbia of that name, as amended or replaced from time to time, and includes any enactment made under the\r\n      authority of that statute or regulation.\r\n    </p>\r\n\r\n  </li>\r\n</ol>\r\n");
```

Comment:
could you re-generate this migration? you seem to have some issues with line endings here

Reply:
Hmmm, it's odd that this is here. This is what the file was supposed to look like
```
using Microsoft.EntityFrameworkCore.Migrations;

namespace Prime.Migrations
{
    public partial class ChangedBCSCReference : Migration
    {
        protected override void Up(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.UpdateData(
                table: "StatusReasonLookup",
                keyColumn: "Code",
                keyValue: 17,
                column: "Name",
                value: "No address from BC Service Card. Enrollee entered address.");
        }

        protected override void Down(MigrationBuilder migrationBuilder)
        {
            migrationBuilder.UpdateData(
                table: "StatusReasonLookup",
                keyColumn: "Code",
                keyValue: 17,
                column: "Name",
                value: "No address from BCSC. Enrollee entered address.");
        }
    }
}
```

Reply:
Not odd, but I don't understand why/how that change happened.